.github/workflows/test.yml

```diff
@@ -139,7 +139,6 @@ jobs:
       # Generate mocks
       - name: Generate mocks
         run: go generate ./...
-        if: ${{ matrix.suite.name != 'Unit' }}
```

Comment:
Why removing this?

Reply:
Since we must call GetRepositoryInfo in a new place in the code, and this code is being tested in unit test, we must run 'go generate'  in order to generate the required files for those tests